src/plugins/share/public/components/tabs/export/export_content.tsx

```diff
@@ -0,0 +1,277 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { useCallback, useEffect, useState } from 'react';
+import { FormattedMessage, injectI18n } from '@kbn/i18n-react';
+
+import {
+  EuiButton,
+  EuiButtonEmpty,
+  EuiCopy,
+  EuiFlexGroup,
+  EuiForm,
+  EuiIcon,
+  EuiModalFooter,
+  EuiRadioGroup,
+  EuiSpacer,
+  EuiSwitch,
+  EuiSwitchEvent,
+  EuiText,
+  EuiToolTip,
+} from '@elastic/eui';
+import useMountedState from 'react-use/lib/useMountedState';
+import { ShareMenuItem } from '../../../types';
+import { type IShareContext } from '../../context';
+
+type ExportProps = Pick<
+  IShareContext,
+  'isDirty' | 'objectId' | 'objectType' | 'onClose' | 'toasts'
+> & {
+  layoutOption?: 'print';
+  aggregateReportTypes: ShareMenuItem[];
+};
```

Comment:
`toasts` is not used
```suggestion
type ExportProps = Pick<IShareContext, 'isDirty' | 'objectId' | 'objectType' | 'onClose'> & {
  layoutOption?: 'print';
  aggregateReportTypes: ShareMenuItem[];
};
```